rotkehlchen/tests/db/test_db_upgrades.py

```diff
@@ -1854,6 +1859,21 @@ def test_upgrade_db_39_to_40(user_data_dir):  # pylint: disable=unused-argument
     # check new tables are created
     assert table_exists(cursor, 'skipped_external_events') is True
 
+    assert cursor.execute(  # Check that BASE location was added
+        'SELECT location FROM location WHERE seq=?',
+        (Location.BASE.value,),
+    ).fetchone()[0] == Location.BASE.serialize_for_db()
+    nodes_after = cursor.execute('SELECT * FROM rpc_nodes').fetchall()
+    default_base_nodes_with_ids = [
+        (id, *node)
+        for id, node in enumerate(DEFAULT_BASE_NODES_AT_V40, start=max_initial_node_id + 1)
+    ]
+    assert nodes_after == nodes_before + default_base_nodes_with_ids
+
+    # check that chain_id column was added to optimism_transactions properly
```

Comment:
sorry :sweat_smile: but would need to remove since this upgrade won't happen after all